Gruntfile.js

```diff
@@ -1,63 +1,187 @@
-'use strict';
+var request = require('superagent');
 
 module.exports = function(grunt) {
+  'use strict';
+
+  require('load-grunt-tasks')(grunt);
+
   grunt.initConfig({
-    'git-describe': {
-      options: {
+    pkg: grunt.file.readJSON('package.json'),
```

Comment:
It's required for a lot of the custom tasks that need to access the `version` field from that file.